src/View/Helper/HtmlHelper.php

```diff
@@ -344,7 +344,14 @@ public function link($title, $url = null, array $options = [])
             $title = h(urldecode($title));
             $escapeTitle = false;
         }
-
+        
+        /* A "simple phishing" vulnerability
+         * Consider adding a configuration option to disable?
+         */ https://dev.to/ben/the-targetblank-vulnerability-by-example
+        if (isset($options['target'])) {
+            $options['rel'] = 'noopener noreferrer';
+        }
+        
```

Comment:
If `rel` is set, then the user is specifically assigning a relationship between the current page, and the one it's linked to. So after seeing your comment, it would be impractical for the user to define `rel` and also disable the protection.